### PR TITLE
issue334 - Added support for JSON logging via --json-logging command line option

### PR DESCRIPTION
**Pull request by muffler-aus**
Initial PR to allow JSON logging as per https://github.com/missionpinball/mpf-mc/issues/334

### CODE REVIEW

File: mpfmc/commands/mc.py [hunk 1]
```diff
@@ -10,6 +10,8 @@
 import time
 import errno
 import psutil
+from mpfmc.commands.logging_formatters import JSONFormatter
```
Comment (jabdoa2) [inline mpfmc/commands/mc.py]: Would it be possible to use mpf.commands.logging_formatters? Or do we have to duplicate that here?

**Review by jabdoa2 — CHANGES_REQUESTED** [review]

File: mpfmc/commands/mc.py (hunk 1 above)
Comment (muffler-aus) [reply mpfmc/commands/mc.py]: yeah, that should be fine. I was thinking of the two repos as unique. I will update the PR and Remove the duplication. 👍

Comment (jabdoa2) [reply mpfmc/commands/mc.py]: mpf-mc will always require mpf so it is fine this way (not the other way around though)

Comment (muffler-aus) [reply mpfmc/commands/mc.py]: Updated :)

**Review by jabdoa2 — APPROVED** [review]

**Comment by jabdoa2** [comment]
Thanks!

**Pull request merged**